nixos/modules/installer/cd-dvd/installation-cd-minimal.nix

```diff
@@ -1,14 +1,17 @@
 # This module defines a small NixOS installation CD.  It does not
 # contain any graphical stuff.
 
-{ ... }:
+{ lib, ... }:
 
 {
-  imports =
-    [ ./installation-cd-base.nix
-    ];
+  imports = [
+    ../../profiles/minimal.nix
+    ./installation-cd-base.nix
+  ];
 
-  isoImage.edition = "minimal";
+  documentation.man.enable = lib.mkForce true;
```

Comment:
```suggestion
  documentation.man.enable = lib.mkDefault true;
```

Reply:
Doing so will conflict with the setting in `nixos/modules/installer/netboot/netboot.nix`

Reply:
Replaced to `lib.mkOverride 500`